recipes/fisspy/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "fisspy" %}
+{% set version = "0.9.61" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b80ba11af1916e06e83d411909c6c5a7fa7e5060b2f6fa2cad2f90f6a6a29869
+
+build:
+  noarch: python
+  number: 2
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - scipy
+    - interpolation
+    - astropy
+    - sunpy
+    - numba
+    - matplotlib
+    - ffmpeg
+    - pillow
+    - numpy
+    - statsmodels
+
+test:
+  imports:
+    - fisspy
+    - fisspy.read
+    - fisspy.analysis
+
+about:
+  home: http://fiss.snu.ac.kr/
+  license: BSD 2-Clause
```

The SPDX identifier has another dash:
```suggestion
  license: BSD-2-Clause
```

---

There also needs to be a licence file